modules/kargoBidAdapter.js

```diff
@@ -76,8 +98,8 @@ export const spec = {
 
     const encodedParams = encodeURIComponent(JSON.stringify(transformedParams));
     return Object.assign({}, bidderRequest, {
-      method: 'GET',
-      url: `${HOST}/api/v2/bid`,
+      method: 'POST',
+      url: `${HOST}/api/v1/prebid`,
       data: `json=${encodedParams}`,
```

Comment:
The payload will no longer need to have `json=` in front of it since it's just a POST request, and it won't need to be encoded since it's not being passed in a URL.